Dfe.Academies.Academisation.Service/Mappers/Legacy/ProjectAggregate/LegacyProjectDetailsMapper.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Dfe.Academies.Academisation.Domain.Core.ProjectAggregate;
+using Dfe.Academies.Academisation.IService.ServiceModels.Legacy.ProjectAggregate;
+
+namespace Dfe.Academies.Academisation.Service.Mappers.Legacy.ProjectAggregate;
+
+internal static class LegacyProjectDetailsMapper
+{
+	internal static ProjectDetails ToDomain(this LegacyProjectServiceModel serviceModel)
+	{
+		return new(1, 1)
```

Comment:
Must use the Urn from the ServiceModel here, only Laestab and Ukprn should be removed. (although I reckon we should validate in the domain that the Urn has not been changed, that's probably not a field that should be user-editable)

Reply:
Done. As you say we can take out Ukprn and Laestab later.